build/readme-parser.ts

```diff
@@ -0,0 +1,65 @@
+/// <reference types="../source/globals" />
```

Comment:
Unrelated question. Why is this needed?

Reply:
Don't ask me!

I think I tried moving `globals` to the root and use it there but it didn't work…